src/Resources/views/ContactForm/ContactFormShopManagerEmail.html.twig

```diff
@@ -1,5 +1,5 @@
 {% block subject %}
-    {{ 'mango_sylius.contactForm.subject.shopManager'|trans({'%customerName%': contact.senderName}) }}
+    {{ 'mango_sylius.contactForm.subject.shopManager'|trans({'%CustomerName%': contact.senderName}) }}
```

Comment:
Always use a lowercase letter in variables

Reply:
DONE